### PR TITLE
Add new playbooks and minor template improvements

### PR DESCRIPTION
**Pull request by nikosev**
This PR introduces the following changes:

- Add logrotate role in saml2samlservers playbook
- Add new playbooks for each SSP instance
- Add roles and tags in existing playbooks
- Style changes in templates
- Add `sources_raw` variable to metarefresh template

### CODE REVIEW

File: d4science2samlservers.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# file: d4science2samlservers.yml
```
Comment (NicolasLiampotis) [inline d4science2samlservers.yml:1]: Investigate whether this can be renamed to keycloak2samlservers.yml

File: deep2samlservers.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# file: deep2samlservers.yml
```
Comment (NicolasLiampotis) [inline deep2samlservers.yml:1]: Investigate whether this can be renamed to something more generic

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: webservers.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@
   roles:
     - { role: apache, tags: apache }
     - { role: php, tags: php }
+    # - { role: egi-igtf, tags: egi-igtf }
```
[the remark this answers is not in the document]
Comment (nikosev) [reply webservers.yml]: done

File: saml2samlservers.yml [hunk 4]
```diff
@@ -9,3 +9,4 @@
     - { role: ssp, tags: ssp }
     #- { role: ssp-module-proxystatistics, tags: statistics }
     #- { role: ssp-modules, tags: ssp-modules }
+    - { role: arillso.logrotate, tags: logrotate }
```
[the remark this answers is not in the document]
Comment (nikosev) [reply saml2samlservers.yml]: done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**